docs/earthly-command/earthly-command.md

```diff
@@ -535,219 +698,359 @@ The token will expire in 1 year from creation date unless a different date is su
 It is then possible to `export EARTHLY_TOKEN=...`, which will force earthly to use this token for all authentication (overriding any other currently-logged in sessions).
 {% endhint %}
 
-#### earthly account remove-token
+#### Options
+
+##### `--write`
 
-###### Synopsis
+Grant write permissions in addition to read permissions
+
+##### `--expiry`
+
+Set token expiry date in the form YYYY-MM-DD or never
+
+Defaults to one (1) year.
```

Comment:
these tokens no longer default to expiring: https://github.com/earthly/earthly/pull/3098